x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/OptimizerRulesTests.java

```diff
@@ -495,7 +495,7 @@ public void testPropagateEquals_VarEq2OrVarNeq5() {
     // a = 2 OR 3 < a < 4 OR a > 2 OR a!= 2 -> TRUE
     public void testPropagateEquals_VarEq2OrVarRangeGt3Lt4OrVarGt2OrVarNe2() {
         FieldAttribute fa = getFieldAttribute();
-        org.elasticsearch.xpack.ql.expression.predicate.operator.comparison.Equals eq = equalsOf(fa, TWO);
```

Comment:
Missed this when I refactored the Optimizer rules, but it didn't break anything until I changed the inheritance in this PR